pkg/operator/ceph/version/version.go

```diff
@@ -86,9 +91,11 @@ func (v *CephVersion) ReleaseName() string {
 
 // ExtractCephVersion extracts the major, minor and extra digit of a Ceph release
 func ExtractCephVersion(src string) (*CephVersion, error) {
+	var build int
 	m := versionPattern.FindStringSubmatch(src)
-	if m == nil {
-		return nil, fmt.Errorf("failed to parse version from: %s", src)
+	mm := buildVersionPattern.FindStringSubmatch(src)
```

Comment:
nit: since we're only using `mm` for the build number, how about moving this line down to line 115 next to where it is used? This will make it more obvious that it is optional IMO.